### PR TITLE
Implementing densenet121

### PR DESCRIPTION
**Pull request by rasha-salim**
This is my initial implementation for the Intel image classification dataset

### CODE REVIEW

**Comment by rasha-salim** [comment]
I implemented densenet pretrained model with learning rate = 0,01, dropout=0.5 and got 85% accuracy.
Still I think we can have better reults with better adjustments of the hyper parameters
